### PR TITLE
fixed all markdown issues

### LINKED ISSUE #613: All markdown issues
### Description:

Message rendering is not done properly

### Expected behavior:

Message should be rendered properly.

### Actual behavior:

https://github.com/user-attachments/assets/63cbd40b-6fff-4aa7-846b-3f9bb6a74e67

### PR DESCRIPTION
**Pull request by smritidoneria**
# Brief Title
Fixed all the message rendering issues.
## Acceptance Criteria fulfillment

- [ ] Task 1
- [ ] Task 2
- [ ] Task 3

Fixes #613 

## Video/Screenshots

https://github.com/user-attachments/assets/2f39a9ce-938c-4466-be29-a219728b1116

### CODE REVIEW

**Comment by devanshkansagra** [comment]
@smritidoneria, Could you please check  and share the result of strikethrough and other markdowns whether it is working or not when the quoted message is sent? When I saw your code, you have made no changes in attachment preview dir. So I think at that point there could be this small bug

**Comment by smritidoneria** [comment]
hey @devanshkansagra , thanks for pointing out, I will look after it

**Comment by Spiral-Memory** [comment]
Hey @smritidoneria 
Could you please resolve the conflicts?

**Comment by smritidoneria** [comment]
hey @Spiral-Memory , changes have been made to resolve the conflicts, kindly look at it.

**Comment by Spiral-Memory** [comment]
Format check is failing @smritidoneria

**Comment by smritidoneria** [comment]
hey @Spiral-Memory ,  Something is wrong with the built
 SyntaxError: Unexpected identifier 'assert'
    at compileSourceTextModule (node:internal/modules/esm/utils:337:16)
    at ModuleLoader.moduleStrategy (node:internal/modules/esm/translators:166:18)
    at callTranslator (node:internal/modules/esm/loader:416:14)
    at ModuleLoader.moduleProvider (node:internal/modules/esm/loader:422:30)
    at link (node:internal/modules/esm/module_job:88:21)
    
    
 This error is coming, 
 
 I have ran prettier, installed babel for the  Transpilation also , but it remains the same.

**Comment by Spiral-Memory** [comment]
Hey @smritidoneria,

It's working for all new PRs except yours. Could you please check if there's any issue with your setup?

If the problem still exists, I'll clone your code and investigate what might be causing it.

**Comment by smritidoneria** [comment]
can u re-run the checks, I have resolved the errors @Spiral-Memory

**Comment by Spiral-Memory** [comment]
Format check passed, lint check failing

**Comment by smritidoneria** [comment]
fixed.

**Review by Spiral-Memory — APPROVED** [review]

**Comment by Spiral-Memory** [comment]
Can you please list all the places you fixed the markdown ? I need to verify

**Comment by smritidoneria** [comment]
while quoting messages and in the delete modal.

File: packages/react/src/views/ChatHeader/ChatHeader.js [hunk 1]
```diff
@@ -385,6 +385,7 @@ const ChatHeader = ({
       </Box>
       {isThreadOpen && (
         <DynamicHeader
+          Markdown
```
Comment (Spiral-Memory) [inline packages/react/src/views/ChatHeader/ChatHeader.js:388]: Passing markdown, and that too seems off ? Why ?

**Review by Spiral-Memory — COMMENTED** [review]
I think this is a mistake.. Why Markdown here ?

File: packages/react/src/views/ChatInput/ChatInput.js [hunk 2]
```diff
@@ -155,7 +156,7 @@ const ChatInput = ({ scrollToBottom }) => {
       messageRef.current.value =
         editMessage.attachments[0]?.description || editMessage.msg;
     } else if (editMessage.msg) {
-      messageRef.current.value = editMessage.msg;
+      messageRef.current.value = <Markdown body={editMessage} />;
```
Comment (Spiral-Memory) [inline packages/react/src/views/ChatInput/ChatInput.js:159]: image

File: packages/react/src/views/ChatHeader/ChatHeader.js (hunk 1 above)
Comment (Spiral-Memory) [inline packages/react/src/views/ChatHeader/ChatHeader.js:388]: I think this is a mistake.. Why Markdown here ?

**Comment by smritidoneria** [comment]
hey @Spiral-Memory , resolved the changes that you have mentioned. kindly look into it.

File: packages/react/src/views/QuoteMessage/QuoteMessage.js [hunk 3]
```diff
@@ -53,62 +54,13 @@ const QuoteMessage = ({ className = '', style = {}, message }) => {
         <Box>{format(new Date(message.ts), 'h:mm a')}</Box>
       </Box>
       <Box css={styles.message}>
-        {message.file ? (
```
Comment (Spiral-Memory) [inline packages/react/src/views/QuoteMessage/QuoteMessage.js:56]: Any specific reason, you are removing these ?

Comment (smritidoneria) [reply packages/react/src/views/QuoteMessage/QuoteMessage.js]: sorry, I mistakenly removed the code

File: packages/react/src/views/ChatHeader/ChatHeader.js [hunk 4]
```diff
@@ -389,6 +389,7 @@ const ChatHeader = ({
...
```
Comment (Spiral-Memory) [inline packages/react/src/views/ChatHeader/ChatHeader.js:392]: @smritidoneria 
Why is Markdown written here ? Looks invalid

**Pull request merged**